homework-online-store/js/main.js

```diff
@@ -70,58 +62,56 @@ function checkStock(stock) {
   return stockEl;
 }
 
-function makeProductCardInCart(product) {
-  const productCardEl = document.createElement('div');
-  productCardEl.classList.add('js-cart__list__item');
-  productCardEl.innerHTML = `
-    <hr>
-    <img src="${product.extra_data.image}" alt="${product.title}">
-    <div class="js-cart__list__info">
-      <h4>${product.extra_data.brand} ${product.extra_data.type}</h4>
-      <span>${product.title}</span><br>
-      <span>Price: $ ${product.price}</span>
-    </div>
-  `; // <button class="js-cart__list__item__delete">delete</button>
-  return productCardEl;
+function isNull(stock) {
+  if (stock == 0) return true;
 }
 
-function displayProduct(product) {
+function toggleVisible(element) {
+  element.classList.toggle('hidden');
+}
+
+function togglePlusMinus(element) {
+  element.classList.toggle('plus');
+  element.classList.toggle('minus');
+}
+
+function displayProductCard(product) {
   const productItemEl = document.createElement('li');
+  productItemEl.id = product.sku;
   productItemEl.innerHTML = makeProductCard(product);
   goodsListEl.appendChild(productItemEl);
   const addToCartBtnEl = productItemEl.querySelector('.js-product-card__cart-button');
   addToCartBtnEl.addEventListener('click', () => {
-    if (addToCartBtnEl.classList.contains('plus')) {
+    if (addToCartBtnEl.classList.contains('plus'))
```

Comment:
I would not rely on the state of the HTML to determine things in my business logic. It will be more clear when we start using React that the (HTML / user interface) is a representation of the state of our data not the other way around. This is fine for now and it works but in the future lets think of other ways.

Reply:
good point, thanks! changed it for checking whether product is in cart or not